docs/source/using/experimental-features.adoc

```diff
@@ -87,3 +87,31 @@ PS> minishift.exe start `
 Be sure to specify a valid gateway and nameserver.
 Failing to do so will result in connectivity issues.
 ====
+
+
+[[set-fixed-ip]]
+== Set fixed IP Address
+
+While the VM gets assigned an IP address using DHCP, most hypervisors do not handle extending the lease time of the assignment.
+This can cause issues on restarts which assigns a new IP address to the instance. While the VM will start, it will conflict
+with the security certificates that have been generated before and the deployment will fail.
+
+For this purpose, we have added a functionality which can set a static address to the VM. This will prevent it from changing
+between restarts. However, it will not work on all the driver plugins at the moment due to the way the IP address is resolved.
+
```

Comment:
To address the issue this feature is added which can set a static IP to the VM. This will prevent the IP to change between {Projcct} restarts. However it will not work with all the docker machine driver plugin at this moment because they handle IP address resolution differently.